src/paypal/ppcp_apple/ppcpOnShippingContactSelectedApple.ts

```diff
@@ -12,17 +12,18 @@ import {
     getCurrency,
     getShipping,
     getShippingLines,
-    setTaxes
+    setTaxes,
+    estimateShippingLines,
+    estimateTaxes,
+    getOrderInitialData
 } from '@boldcommerce/checkout-frontend-library';
 
 export async function ppcpOnShippingContactSelectedApple(event: ApplePayShippingContactSelectedEvent): Promise<void> {
     const {iso_code: currencyCode} = getCurrency();
     const applePaySession = getPPCPApplePaySessionChecked();
     const address = formatApplePayContactToCheckoutAddress(event.shippingContact);
-    address.first_name = address.first_name.trim() || 'fistName';
-    address.last_name = address.last_name.trim() || 'lastName';
-    address.address_line_1 = address.address_line_1.trim() || 'addressLine1';
-    address.phone_number = address.phone_number.trim() || '0000000000';
+    const {rsa_enabled: rsaEnabled} = getOrderInitialData();
```

Comment:
I think you can use  the `rsa_enabled` from `general_settings`, so we don't need to update front end library
```
const {general_settings} = getOrderInitialData();
const rsaEnabled = general_settings.checkout_process.rsa_enabled;
```

Reply:
Thanks for the info, I didn't know it was available there. i'll add that in

Reply:
Added, and I'll close the frontend library change